easybuild/easyconfigs/l/libssh/libssh-0.9.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'CMakeMake'
+
+name = 'libssh'
+version = '0.9.0'
+
+homepage = 'https://www.libssh.org'
+
+description = """Multiplatform C library implementing the SSHv2 protocol on client and server side"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.libssh.org/files/0.9/']
+sources = ['%(name)s-%(version)s.tar.xz']
+checksums = ['25303c2995e663cd169fdd902bae88106f48242d7e96311d74f812023482c7a5']
+
+builddependencies = [
+    ('CMake', '3.12.1'),
+    ('binutils', '2.28'),
+]
+
```

Comment:
At the very least, this probably has a OS dependency on OpenSSL?

Reply:
So, this should either add a dep. on OpenSSL or add a os dependency on it.

Reply:
@SimonPinches please add an os dependency for SSL, as @Micket requested?
```
osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```

Reply:
Done - apologies for the delay!